webapp-django/crashstats/supersearch/models.py

```diff
@@ -212,11 +175,21 @@ class SuperSearchField(models.SocorroMiddleware):
     def get(self, **kwargs):
         raise NotImplemented()
 
+    def create_field(self, **kwargs):
+        # print "IMPL", self.get_implementation()
+        return self.get_implementation().create_field(**kwargs)
+
+    def update_field(self, **kwargs):
+        return self.get_implementation().update_field(**kwargs)
+
+    def delete_field(self, **kwargs):
+        return self.get_implementation().delete_field(**kwargs)
+
     def post(self, payload):
-        return super(SuperSearchField, self).post(self.URL_PREFIX, payload)
+        raise NotImplementedError('Use create_field')
```

Comment:
Wouldn't it be simpler to just remove those 2 methods? It will raise a pretty similar error anyway.